.rubocop.yml

```diff
@@ -101,6 +101,10 @@ Style/Copyright:
 Style/HashSyntax:
   EnforcedStyle: ruby19_no_mixed_keys
 
+Style/NumericLiterals:
+  AllowedPatterns:
+    - \d{4}_\d{2}_\d{2}_\d{6}
```

Comment:
I thought it would be good to have a comment on what the pattern means (e.g. YYYY_MM_DD_hhmmss)

Reply:
Good point, I think I had that when I first was playing around with this, but didn't use the stash where I had that. I'll make that change when I rebase this for the conflict